_data/navbars/maintenance.yml

```diff
@@ -31,6 +31,20 @@ section:
     path: /maintenance/ebpf/ebpf-and-eks
   - title: Troubleshooting
     path: /maintenance/troubleshoot/troubleshoot-ebpf
+- title: calicoctl
+  path: /maintenance/clis/calicoctl/
+  section:
+  - title: Install calicoctl
+    path: /maintenance/clis/calicoctl/install
+  - title: Configure calicoctl
+    path: /maintenance/clis/calicoctl/configure/
+    section:
+    - title: Overview
+      path: /maintenanceclis/calicoctl/configure/overview
```

Comment:
I think you're missing a `/` in the path here - `maintenanceclis` should be `maintenance/clis`

Reply:
I fixed that just now, rebased again. Still not catching the conflicts. If you go to this page in netlify: https://deploy-preview-4705--calico-master.netlify.app/getting-started/clis/calicoctl/install
and click "Install and configure calicoctl" link, it is 404.

Reply:
That link looks correct to me - the place you linked to no longer exists because we moved the files out of `getting-started`, so I would expect it to be a 404.

All the links within `maintenance/` seem to be working though.

Also, I wouldn't expect any more conflicts on this PR.

Reply:
Sorry, mixing "conflicts" with broken links.